src/Example.php

```diff
@@ -1,11 +1,18 @@
 <?php
 
+declare(strict_types=1);
```

Comment:
This is super important. Without this, this PHP application will be considered as wasted. Because everyone was waiting for this strict type in PHP and in PHP 7.4 or greater brought this and this will make the whole PHP application bullet proof, error-prone at least during runtime.

Without this strict type, now a days, PHP app is not a even an application. You must need to use strict type in PHP.